### PR TITLE
Fix  #3959

### LINKED ISSUE #3959: Bug: Guns can go into grinders
https://discordapp.com/channels/255035529085583360/375588651599396866/632406944535281675

### PR DESCRIPTION
**Pull request by AlexMorgan3817**
## About The Pull Request
<details><summary>Fix #3959</summary>
Now you can't put pistol or anything without reagents in makeshift grinder
</details>

### CODE REVIEW

**Comment by AlexMorgan3817** [comment]
<div title="Ready for Review">R4R</div>

File: code/modules/reagents/machinery/grinder.dm [hunk 1]
```diff
@@ -372,7 +372,7 @@
 		if(!container.standard_pour_into(user, src))
 			. = ..()
 	else
-		. = ..()
+		to_chat(user, SPAN_NOTICE("\icon[I] \the [I] seems to you that it is not suitable for a [src] \icon[src]"))
```
Comment (SpadesSlick) [inline code/modules/reagents/machinery/grinder.dm]: I think it should be worded better.

**Comment by AlexMorgan3817** [comment]
R4R

File: code/modules/reagents/machinery/grinder.dm [hunk 2]
```diff
@@ -372,7 +372,7 @@
 		if(!container.standard_pour_into(user, src))
 			. = ..()
 	else
```
Comment (comma) [inline code/modules/reagents/machinery/grinder.dm]: I don't see where it checks for if object has reagents at all, it just prevents putting ANY object in as far as I can tell.
Which will break the grinding part of grinder.
Add `else if (LAZYLEN(O.reagents) . = ..()` before this branch maybe?

Comment (drexample) [reply code/modules/reagents/machinery/grinder.dm]: ^

**Comment by AlexMorgan3817** [comment]
ready4review

**Pull request merged**